packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -592,25 +599,53 @@ export function MainStackScreen() {
       pincodeType,
       account,
       hasSeenVerificationNux,
+      shouldSkipOnboardingEducationScreen: savedShouldSkipEducationScreen,
     } = mapStateToProps(store.getState())
 
+    // Remove Onboarding Education Screen Experiment: Because remote configs are fetched after the initial route is launched,
+    // The randomization is hardcoded by device id here to achieve a 50/50 split, the value is written into the redux store so the same experience
+    // would persist. This block of code should be removed when the experiment is done.
+    const rng = seedrandom('deviceInfoModule.getUniqueId()')
```

Comment:
Is this a mistake?  It looks like just a hardcoded string instead of a function call